### PR TITLE
Backend User management

### PR DESCRIPTION
**Pull request by haonh**

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class Admin::UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+  def index
```
Comment (ThuBM) [inline app/controllers/admin/users_controller.rb]: add space before this line

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    @user.destroy
+    flash[:success] = t "destroy_message"
+    redirect_to admin_users_path	
+  end
+
+  private
+    def set_user
+      @user = User.find params[:id]
+    end
+    def user_params
```
Comment (ThuBM) [inline app/controllers/admin/users_controller.rb]: add  new line

File: app/views/admin/users/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        <% end %>
+      </td>
+      <td>
+        <%= link_to "" , admin_user_path(user), method: :delete,
+          data: {confirm: t("confirm", action: t("delete"))},
+          class: "glyphicon glyphicon-remove" %>
+      </td>
+  <% end %>
+</table>
+<%= will_paginate @users %>
```
Comment (ThuBM) [inline app/views/admin/users/index.html.erb]: remove @users

File: app/assets/stylesheets/custom.css.scss [hunk 4]
```diff
@@ -146,3 +146,6 @@ input {
 .admin_interface_style{
   font-size: 25px;
 }
+.small-button{
```
Comment (ThuBM) [inline app/assets/stylesheets/custom.css.scss]: new line before this line

**Comment by ThuBM** [comment]
OK

File: app/controllers/admin/users_controller.rb [hunk 5]
```diff
@@ -0,0 +1,28 @@
+class Admin::UsersController < ApplicationController
+  before_action :init_user, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @users = User.paginate page:params[:id], per_page: Settings.paginate_per_page
+  end
+
+  def update
+    @user.update_attributes user_params
```
Comment (chuanhtuan) [inline app/controllers/admin/users_controller.rb]: if fail

**Pull request merged**